src/briefcase/bootstraps/toga.py

```diff
@@ -118,7 +118,7 @@ def pyproject_table_linux_system_suse(self):
     # Needed to provide GTK
     "gtk3",
     # Needed to support Python bindings to GTK
-    "gobject-introspection", "typelib(Gtk)=3.0",
+    "gobject-introspection", "typelib(Gtk) = 3.0",
```

Comment:
I don't really understand all this notation....but I finally got `zypper` to understand it with these spaces...